src/Lexer.js

```diff
@@ -75,6 +78,8 @@ module.exports = class Lexer {
     this.tokenizer.rules = rules;
   }
 
+  static staticLexer
```

Comment:
I don't think this static property is necessary since `Lexer.staticLexer` will be `undefined` without this line until it is set anyways.

Reply:
Fixed.